src/Mod/Fem/feminout/readFenicsXML.py

```diff
@@ -244,7 +244,7 @@ def invertdict(dic):
         for (elm, numbers) in list(element_dict.items()):
             lst = sorted(list(numbers.items()), key=lambda x: x[0])
             if lst != []:
-                Console.PrintWarning(elm, " min: ", lst[0], " max: ", lst[-1], "\n")
+                Console.PrintWarning("{} min: {} max: {}\n".format(elm, lst[0], lst[-1]))
```

Comment:
```suggestion
                Console.PrintWarning(f"{elm} min: {lst[0]} max: {lst[-1]}\n")
```
Make it even better!

Reply:
@luzpaz Thanks for the heads up. It's a little late, but I will mark it as solved :-)